projects/igniteui-angular/src/lib/grids/cell.component.html

```diff
@@ -4,7 +4,7 @@
 <ng-template #defaultCell>
     <div igxTextHighlight style="pointer-events: none" [cssClass]="highlightClass" [activeCssClass]="activeHighlightClass" [groupName]="gridID"
         [value]="formatter ? formatter(value) : column.dataType === 'number' ? (value | igxdecimal: grid.locale) : column.dataType === 'date' ? (value | igxdate: grid.locale) : value"
-        [row]="rowData" [column]="this.column.field" [containerClass]="'igx-grid__td-text'"
+        [row]="rowData" [column]="this.column.field" [containerClass]="'igx-grid__td-text'" [metadata]="searchMetadata"
```

Comment:
This change should be made to the tree-cell as well.

Reply:
Also to the expandable cell.